### DISCUSSION TITLE
bS-Isotope question

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#523] [Q&A]
Hello dears!

im trying to display my posts (regular posts) to display by category. 
Can you suggest me how to archive this or even if its possible with your plugin?

Thank you so much. <3

**Comment by crftwrk** [comment]
Hi Richard,

I do not know what you exactly mean, but as described here https://bootscore.me/documentation/plugin/bs-isotope/#Selecting_the_post_type you can show any post by category. Categories you want to show must wrapped in a parent category. Use the id of the parent category in the shortcode:

```
[bs-isotope-equal-height type="post" tax="category" cat_parent="YOUR ID"]
```

Replace YOUR ID with parent category id.

Solved?

**Reply by ritchiemarkov** [reply]
Hello, sorry for annoying!

I have 3 types of wordpress post category (travels, maps, and galleries).

so now, whats should i perform in order to visualize isotope with those 3 wp post categories?

I swear im confused.

**Reply by crftwrk** [reply]
Filtering is always done by the id of the parent-category. So, what you have to do is to wrap those 3 categories in a parent-catergory:

- Blog (parent-category, use this id in the shortcode)
   - Travels
   - Maps
   - Galleries
